src/components/side_panel/chart/main_chart_panel/fill_color_picker_entry.xml

```diff
@@ -0,0 +1,20 @@
+<templates>
+  <t t-name="o-spreadsheet-FillColorPickerEntry" owl="1">
+    <div class="o-tools" style="display: inline-block;">
+      <div class="o-tool o-dropdown o-with-color">
```

Comment:
We should definetly use a dedicated component everywhere we use a colorPicker and not only in the chart side panel. 
If there is the need for different style, just add a props `style` or `class` and use it to have different behaviour in the side panels/topbar. 

Side note but I think we want a `cursor: pointer` on hover in the side panel too.